package.json

```diff
@@ -1,25 +1,25 @@
 {
   "name": "@lambdacurry/component-library",
-  "version": "1.14.1",
+  "version": "1.14.2-beta.0",
   "main": "dist/index.js",
-  "module": "dist/component-library.esm.js",
+  "module": "dist/index.js",
   "types": "dist/index.d.ts",
+  "sideEffects": [
+    "./src/lib/charts/BarChart/roundedBarCharts.ts"
```

Comment:
What does this mean?

Reply:
It tells webpack what files contain sideEffects, so they can't be tree-shaken:

https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free